p2p.go

```diff
@@ -40,61 +37,33 @@ type HandleUnicast func(ctx context.Context, w io.Writer, data []byte) error
 
 // Config enumerates the configs required by a host
 type Config struct {
-	HostName                 string          `yaml:"hostName"`
-	Port                     int             `yaml:"port"`
-	ExternalHostName         string          `yaml:"externalHostName"`
-	ExternalPort             int             `yaml:"externalPort"`
-	SecureIO                 bool            `yaml:"secureIO"`
-	Gossip                   bool            `yaml:"gossip"`
-	ConnectTimeout           time.Duration   `yaml:"connectTimeout"`
-	MasterKey                string          `yaml:"masterKey"`
-	Relay                    string          `yaml:"relay"` // could be `active`, `nat`, `disable`
-	ConnLowWater             int             `yaml:"connLowWater"`
-	ConnHighWater            int             `yaml:"connHighWater"`
-	RateLimiterLRUSize       int             `yaml:"rateLimiterLRUSize"`
-	BlackListLRUSize         int             `yaml:"blackListLRUSize"`
-	BlackListCleanupInterval time.Duration   `yaml:"blackListCleanupInterval"`
-	ConnGracePeriod          time.Duration   `yaml:"connGracePeriod"`
-	EnableRateLimit          bool            `yaml:"enableRateLimit"`
-	RateLimit                RateLimitConfig `yaml:"rateLimit"`
-	PrivateNetworkPSK        string          `yaml:"privateNetworkPSK"`
-}
-
-// RateLimitConfig all numbers are per second value.
-type RateLimitConfig struct {
-	GlobalUnicastAvg   int `yaml:"globalUnicastAvg"`
-	GlobalUnicastBurst int `yaml:"globalUnicastBurst"`
-	PeerAvg            int `yaml:"peerAvg"`
-	PeerBurst          int `yaml:"peerBurst"`
+	HostName          string        `yaml:"hostName"`
+	Port              int           `yaml:"port"`
+	ExternalHostName  string        `yaml:"externalHostName"`
+	ExternalPort      int           `yaml:"externalPort"`
+	SecureIO          bool          `yaml:"secureIO"`
+	Gossip            bool          `yaml:"gossip"`
+	ConnectTimeout    time.Duration `yaml:"connectTimeout"`
+	MasterKey         string        `yaml:"masterKey"`
+	ConnLowWater      int           `yaml:"connLowWater"`
+	ConnHighWater     int           `yaml:"connHighWater"`
+	ConnGracePeriod   time.Duration `yaml:"connGracePeriod"`
+	PrivateNetworkPSK string        `yaml:"privateNetworkPSK"`
```

Comment:
I'm unsure if we need the private network PSK stuff, is it used anywhere in the file?